src/ripple/app/ledger/LedgerEntrySet.h

```diff
@@ -338,6 +347,21 @@ class LedgerEntrySet
 
     bool checkState (SLE::pointer state, bool bSenderHigh,
         Account const& sender, STAmount const& before, STAmount const& after);
+
+    STAmount subtractFTCredits (Account const& main,
+                                Account const& other,
+                                STAmount const& amount) const;
+};
+
+class EnableFundingTableGuard
+{
+private:
+    LedgerEntrySet& les_;
+    bool shouldDisable_{false};
```

Comment:
Trivial: Most other inline member initialization I've seen in rippled has used the assignment form. `bool sholdDisable_ = false;`